multimodal/setup.py

```diff
@@ -32,24 +32,25 @@
     "seqeval<=1.2.2",
     "evaluate<=0.3.0",
     "accelerate>=0.9,<0.14",
+    "tensorboard<2.12.0",
     "timm<0.7.0",
-    "torch>=1.9,<1.13",
-    "torchvision<0.14.0",
-    "torchtext<0.14.0",
+    "torch>=1.9,<1.14",
```

Comment:
You'll need to update tabular's fastai extra_dependencies key since it also requires torch. Maybe we move torch to the `_setup_utils.py` file so we ensure consistent versions across the package?